server/data/mappers/jobComparator.ts

```diff
@@ -24,4 +24,28 @@ const jobComparator = (left: Job, right: Job): number => {
   return 0
 }
 
-export default jobComparator
+/**
+ * Comparator function that compares two `WorkInterestJob` view model instances, returning -1, 0 or 1 depending on whether the
+ * first job's type property is alphabetically before, equal or after the second job's type property.
+ * If the first job's type is 'OTHER' then 1 is returned.
+ *
+ * This comparator function can be used to sort an array of `WorkInterestJob` view model instances, resulting in the array being sorted
+ * alphabetically on the `type` property, except 'OTHER' which will always be at the end of the array.
+ */
+const workInterestJobComparator = (left: WorkInterestJob, right: WorkInterestJob): number => {
```

Comment:
Rather than put this in a separate source file (eg: `workInterestJobComparator.ts`) I opted to put this in `jobComparator.ts` as the 2 functions (`jobComparator` and `workInterestJobComparator` are so closely related. They operate on different types of objects, but essentially they do the same thing, and the object types are both "jobs")